build.sbt

```diff
@@ -63,9 +63,9 @@ lazy val root = (project in file(".")).enablePlugins(PlayScala, RiffRaffArtifact
     packageSummary := "tag manager",
     packageDescription := """manage tags""",
 
-    doc in Compile := (target.value / "none"),
+    Compile / doc := (target.value / "none"),
     scalaVersion := "2.12.13",
-    scalaVersion in ThisBuild := "2.12.13",
+    ThisBuild / scalaVersion := "2.12.13",
```

Comment:
Just mentioning you should plan to update `scala` to `2.13.5` which should be quite straightforward.

Reply:
Good to know. Looks like we'll need a 2.13 build for `panda-hmac-play` in consuming repositories, including this one.